### PR TITLE
#258: Adds federico's description on team (closes #258)

### LINKED ISSUE #258: Adds federico's description on team
## requirements
Federico needs a description on the team page

### PR DESCRIPTION
**Pull request by fes300**
Closes #258

### tests performed

> A Test Plan is used to show what you tested to make sure your code works fine. You should write here all that you did to test, and provide some results of your testing.

> These results can be screenshots, query results, or even just pointers to unit tests that successfully passed.

### tests not performed (domain coverage)

> At times not everything can be tested, and writing what hasn't been tested is just as important as writing what has been tested.

> An example of partial test is a field displaying 4 possible values. If 3 values are tested, with screenshots, and 1 is not, then it should be mentioned here.

### CODE REVIEW

File: src/Team/Who/people.js [hunk 1]
```diff
@@ -226,6 +226,24 @@ const people = [
...
     )
   },
+  {
+    nickname: 'federico',
+    fullName: 'Federico Sordillo',
+    skills: ['software engineer', 'full-stack developer'],
+    bio: (
+      <div>
+        <p>
+          Federico comes from a multidisciplinary background, after a degree in Sociology he switched field and decided to get his specializaition in Economics at University Luigi Bocconi.
```
Comment (ecamellini) [inline src/Team/Who/people.js]: image

Trailing space, build complains 😛 

Suggested extension for VSCode https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces

There is also a VSCode option for that: `"files.trimTrailingWhitespace"`

Comment (ecamellini) [inline src/Team/Who/people.js]: Typo: "specializaition" --> "specialization"

File: src/Team/Who/people.js [hunk 2]
```diff
@@ -226,6 +226,24 @@ const people = [
...
+    nickname: 'federico',
+    fullName: 'Federico Sordillo',
+    skills: ['software engineer', 'full-stack developer'],
+    bio: (
+      <div>
+        <p>
+          Federico comes from a multidisciplinary background, after a degree in Sociology he switched field and decided to get his specializaition in Economics at University Luigi Bocconi. 
+        </p>
+        <p>
+          After working a few years as business / financial analyst he decided to swallow the red pill and since then he spends most of his time exploring the neverending cave of code. He landed at Buildo while searching for a place that allowed him to produce quality code.
```
Comment (ecamellini) [inline src/Team/Who/people.js]: Grammarly suggests a comma after `analyst`

File: src/Team/Who/people.js (hunk 1 above)
Comment (fes300) [reply src/Team/Who/people.js]: yes sorry I skipped the test/lint step! 😭

**Comment by fes300** [comment]
🎉

**Pull request merged**